config/aws-config.js

```diff
@@ -5,20 +5,35 @@ const AWS = require('aws-sdk')
 
 const localstack = process.env.LOCALSTACK || 0
 
-const secretsManagerConfig = localstack ? { endpoint: 'http://localhost:4584', region: 'us-west-2' } : {}
-const systemManagerConfig = localstack ? { endpoint: 'http://localhost:4583', region: 'us-west-2' } : {}
-const stsConfig = localstack ? { endpoint: 'http://localhost:4592', region: 'us-west-2' } : {}
+let secretsManagerConfig = {}
+let systemManagerConfig = {}
+let stsConfig = {}
+
+if (localstack) {
+  secretsManagerConfig = {
+    endpoint: process.env.LOCALSTACK_SM_URL || 'http://localhost:4584',
+    region: process.env.AWS_REGION || 'us-west-2'
+  }
+  systemManagerConfig = {
+    endpoint: process.env.LOCALSTACK_SSM_URL || 'http://localhost:4583',
+    region: process.env.AWS_REGION || 'us-west-2'
+  }
+  stsConfig = {
+    endpoint: process.env.LOCALSTACK_STS_URL || 'http://localhost:4592',
+    region: process.env.AWS_REGION || 'us-west-2'
+  }
+}
 
 module.exports = {
-  secretsManagerFactory: (opts) => {
+  secretsManagerFactory: (opts = {}) => {
     if (localstack) {
-      opts = secretsManagerConfig
+      Object.assign(opts, secretsManagerConfig)
```

Comment:
nit: `opts = Object.assign({}, opts, secretsManagerConfig)`

I worry copying into `opts` directly could lead to bugs later.  *E.g.*, if the caller reuses the `opts` object it passes.